### ISSUE TITLE
Doesn't work at all any more in BetterDiscord

### ISSUE THREAD

**Issue by AstudilloMarillo** [DiscordStyles/HorizontalServerList#42]
### Describe the bug
When I turn this theme on, the server list disappears from the left and does not appear on the top, so is essentially just gone until I turn off the theme.

### To Reproduce  
Run the latest Discord/BetterDiscord, have no other themes/plugins on, turn on the HorizontalServerList theme. 

### Screenshots
not really necessary

### Infomation (please complete the following information)
**Discord channel**: Stable
**OS**: Windows 10
**Mod**: BetterDiscord
**Discord language**: English

**Comment by Gibbu** [comment]
Seems fine for me
<img width="145" alt="image" src="https://github.com/DiscordStyles/HorizontalServerList/assets/20338746/88cc710a-44d5-4832-b571-824543a72e42">

**Comment by YetAnotherDevWannabe** [comment]
Doesn't work for me either on 1.9.4

**Comment by fwenny** [comment]
yep, can confirm, it stopped working after the last Discord update

**Comment by joker-119** [comment]
Can also confirm, this as well as other themes are no longer working, though there's no console errors/issues

**Comment by NathanTheGraves** [comment]
Works again for me with Better Discord v1.9.5 👍

**Comment by AstudilloMarillo** [comment]
> Works again for me with Better Discord v1.9.5 👍

For me it doesn't.
